stats/stats.go

```diff
@@ -422,6 +422,27 @@ func (h *Histogram) copyHDataFrom(src *Histogram) {
 	}
 }
 
+// Merge two different histogram with different scale parameters
+// Highest offset and divider value will be selected on new Histogram to maximize scope
+func Merge(h1 *Histogram, h2 *Histogram) *Histogram {
+	var divider float64
```

Comment:
nit: I think:
```
divider := h1.Divider
if  h2.Divider > h1.Divider {
	divider = h2.Divider
}
```
would be a more go/standard way to write this (quite a lot shorter, gets the right type automatically etc)